functions.php

```diff
@@ -998,9 +998,10 @@ function display_more_stories_featured( $post ) {
 
 function display_related_story( $story ) {
 	$thumbnail = get_the_post_thumbnail_url( $story->ID, 'post-thumbnail', array( 'class' => 'img-responsive' ) );
+	$thumbnail = $thumbnail ?: FEED_THUMBNAIL_FALLBACK;
 	ob_start();
 ?>
-	<div class="span3">
+	<div class="span3 match-height">
```

Comment:
Just in case you're not aware, you can initialize matchHeight without having to define a class for it, via data attributes (`data-mh`): https://github.com/liabru/jquery-match-height#data-api

It may be worth having a dedicated class for it in this theme however, in case matchHeight is ever needed within one of these feature stories (there's a good chance the WP content filters would strip it out if the data attribute were added to post content manually).